Task_DEV_3/Task_DEV_3/Program.cs

```diff
@@ -4,39 +4,49 @@ namespace Task_DEV_3
 {
     class Program
     {
+        static void InputArrayElement(int[] inputNumbers, int arrayLength)
+        {
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+        }
+        static void CheckingNumbersForPrime(int[] inputNumbers, int arrayLength)
```

Comment:
Would be better to create method which checks one number and return `true `or `false`.

Reply:
The first word in method name should be verb.